spanner/api/Spanner/Program.cs

```diff
@@ -1769,23 +1769,44 @@ await connection.BeginTransactionAsync())
                 new Singer {SingerId = 5, FirstName = "David",
                     LastName = "Lomond"},
             };
+            List<Album> albums = new List<Album> {
+                new Album {SingerId = secondSingerId, AlbumId = 1,
```

Comment:
nit: a space between { and first property - here and everywhere

Reply:
Unresolved this because this still needs fixing.